sebastien15/capstoneproject
# Ft delete an article

#### What does this PR do?
>This PR is deletes a single article from mongodb by finding their id

#### Description of Task to be completed
>Added a method to delete an article

#### How should this be manually tested?

- Clone this repo with `git clone git@github.com:sebastien15/myportifolio.git` 
- Checkout to `ft-delete-an-article`
- npm install nodemon -g
- npm install
- npm start
- run localhost:3000/deleteBlog/{id} in post man make sure it is a delete method


#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/UtApCefK
#### Screenshots (if appropriate)
>
![delete blog](https://user-images.githubusercontent.com/41580903/93979536-bfdea400-fd7d-11ea-8975-09bab23a3e04.PNG)
